dice_servers/tripleawarclub-staging.properties

```diff
@@ -0,0 +1,22 @@
+# when placing in the list, sort by order
+order=11
+# name to display to the user
+name=dice.tripleawarclub.org (experimental BETA)
+# dice server host
+host=www.tripleawarclub.org
```

Comment:
As discussed, we probably want to avoid this domain

Reply:
(Just so this isn't being missed accidentally, you know when you think about everything else but not this small thing 😄 )

Reply:
Done in 7ed87cf.